SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -11,38 +11,40 @@ Buffer {
 	*initClass { serverCaches = IdentityDictionary.new }
 
 	// doesn't send
-	*new { arg server, numFrames, numChannels, bufnum;
+	*new { arg server, numFrames, numChannels, bufnum, sampleRate;
 		server = server ? Server.default;
 		bufnum ?? { bufnum = server.nextBufferNumber(1) };
 		^super.newCopyArgs(
 			server,
 			bufnum,
 			numFrames,
-			numChannels
-		).sampleRate_(server.sampleRate).cache
+			numChannels,
+			sampleRate !? _.asFloat ?? 0.0
```

Comment:
I don't understand why there has to be a difference between *new and *alloc in converting the supplied sampleRate. Since the conversion is done when sending messages (allocMsg, setSampleRateMsg), I think we can remove it from here.

Reply:
there shouldn't be so I'll correct